model/registry/db.yaml

```diff
@@ -274,17 +274,24 @@ groups:
         brief: The name of the primary table that the operation is acting upon, including the database name (if applicable).
         note: >
           It is not recommended to attempt any client-side parsing of
-          `db.statement` just to get this property, but it should be set if
+          `db.query.text` just to get this property, but it should be set if
           it is provided by the library being instrumented.
           If the operation is acting upon an anonymous table, or more than one table, this
           value MUST NOT be set.
         examples: ['public.users', 'customers']
         tag: tech-specific-sql
-      - id: statement
+      - id: query.text
         type: string
         stability: experimental
         brief: >
-          The database statement being executed.
+          The database query being executed.
+        examples: ['SELECT * FROM wuser_table', 'SET mykey "WuValue"']
+        tag: db-generic
+      - id: query.parameter
+        type: template[string]
+        stability: experimental
+        brief: >
+          The query parameters, `<key>` being the parameter name (or index number), the value being the parameter value.
```

Comment:
> `<key>` being the parameter name (or index number)

`index number` starting with `0` or `1`? Should we maybe make it here explicit [to avoid confusions like here](https://stackoverflow.com/questions/616135/in-jdbc-why-do-parameter-indexes-for-prepared-statements-begin-at-1-instead-of)?

Reply:
> Should we maybe make it here explicit [to avoid confusions like here](https://stackoverflow.com/questions/616135/in-jdbc-why-do-parameter-indexes-for-prepared-statements-begin-at-1-instead-of)?

yes, definitely